CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ if (${CMAKE_CXX_COMPILER_ID} STREQUAL "MSVC")
   set(RELEASE_FLAGS, "/GS /GL /W3 /Gy /Zi /O2")
   set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 else()
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
```

Comment:
This needs to be combined into `${DEBUG_FLAGS}` and `${RELEASE_FLAGS}`. This way we leave the intrinsic compiler flags alone and stick to configuration specific flags :smile: